### PR TITLE
Feature: Market Profit Loss/Win Simulator Slider

### PR DESCRIPTION
**Pull request by andre-meyer**
@cag Please take a look if I got this right?

### CODE REVIEW

File: app/src/Sidebar_WithScalar/Buy/index.js [hunk 1]
```diff
@@ -74,26 +72,51 @@ const Buy = ({
...
+      const investmentAmountInUnits = collateral.toUnitsMultiplier.mul(
+        hasEnteredInvestment ? investmentAmount : zeroDecimal
+      );
 
       const normalizedSlider = new Decimal(sliderValue)
         .sub(decimalLower)
         .div(decimalUpper.sub(decimalLower));
 
-      const profitAmount = maxPayout.mul(normalizedSlider);
+      const invertSlider = oneDecimal.sub(normalizedSlider);
```
Comment (denisgranha) [inline app/src/Sidebar_WithScalar/Buy/index.js]: what is the purpose of this property? I would add a comment explaining it

File: app/src/Sidebar_WithScalar/Buy/index.js [hunk 2]
```diff
@@ -307,96 +333,109 @@ const Buy = ({
...
+          <>
+            <div>
+              <div className={cx("slider")}>
+                <div className={cx("labels")}>
+                  <span>
+                    {formatScalarValue(market.lowerBound, market.unit)}
+                  </span>
+                  <span>
+                    {formatScalarValue(
+                      (market.upperBound - market.lowerBound) / 2 +
```
Comment (denisgranha) [inline app/src/Sidebar_WithScalar/Buy/index.js]: the odds here looks correct

File: app/src/Sidebar_WithScalar/Buy/index.js [hunk 3]
```diff
@@ -74,26 +73,54 @@ const Buy = ({
...
+      const investmentAmountInUnits = collateral.toUnitsMultiplier.mul(
+        hasEnteredInvestment ? investmentAmount : zeroDecimal
+      );
 
       const normalizedSlider = new Decimal(sliderValue)
         .sub(decimalLower)
         .div(decimalUpper.sub(decimalLower));
 
-      const profitAmount = maxPayout.mul(normalizedSlider);
+      // the inverted version of this slider, going from 1 to 0 instead of 1 to 0, to show the opposite outcome
```
Comment (cag) [inline app/src/Sidebar_WithScalar/Buy/index.js]: "1 to 0 instead of 0 to 1" right? :P

File: app/src/Sidebar_WithScalar/Buy/index.js [hunk 4]
```diff
@@ -307,96 +337,115 @@ const Buy = ({
...
+                  </span>
+                </div>
+              </div>
+            </div>
+            <div className={cx("invest-summary")}>
+              <div className={cx("summary")}>
+                <div className={cx("row")}>
+                  <span className={cx("label")}>Max Payout</span>
+                  <span className={cx("spacer")} />
+                  <span className={cx("value")}>123 DAI</span>
```
Comment (cag) [inline app/src/Sidebar_WithScalar/Buy/index.js]: Wait, is this supposed to be hardcoded?

File: app/src/Sidebar_WithScalar/Buy/index.js (hunk 3 above)
Comment (andre-meyer) [reply app/src/Sidebar_WithScalar/Buy/index.js]: oops right

File: app/src/Sidebar_WithScalar/Buy/index.js (hunk 4 above)
Comment (andre-meyer) [reply app/src/Sidebar_WithScalar/Buy/index.js]: no, it's placeholders

**Pull request merged**